src/components/invoice-list/Buttons/NavigationListButtons.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
musisz sie zastanowic czy to jest komponent reuzywalny dla calej aplikacji czy tylko dla invoice-list (tak wskazuje miejsce gdzie sie znajduje)?
Obecnie implementujesz tutaj obsluge stricte Invoice'a w komponencie, ktorego nazwa nie wskazuje na to, ze obsluguje ona tylko Invoice'a, ktos moglby to przez przypadek zaimportowac chcac uzyc i by sie mocno zdziwil. 
Rozwiazania sa dwa: 
- zmienic nazwe na np InvoiceListNavigation
-  zrobic reuzywalny komponent (jesli jest w ogole taki potrzebny) i wtedy przekazac children z kontentem specyficznym dla danego uzycia, troche takie HOC (highest order component)